mcp/README.md

```
@@ -0,0 +1,92 @@
+
+1. **Hosted MCP Server**
+2. **Local MCP Server** - For integration with Claude Desktop and other MCP clients
+
+## Hosted MCP Server
+
+Go to https://app.honcho.dev and get an API key. Then go to Claude Desktop and navigate to custom MCP servers.
+
+If you don't have node/npm/npx install you will need to do that. Claude Desktop or Claude Code can help!
+
+Add or integrate this into your claude desktop config:
+```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)